docs/topics/auth/default.txt

```diff
@@ -961,11 +961,21 @@ implementation details see :ref:`using-the-views`.
     * ``extra_context``: A dictionary of context data that will be added to the
       default context data passed to the template.
 
+    * ``redirect_authenticated_user``: Boolean. If True, authenticated users
+      accessing the login page will be redirected as if they had just successfully
+      logged in.
+      If the redirect url redirects to the same login page, :exc:`ValueError`
```

Comment:
If this is meant as a separate paragraph, it needs a blank line above it. Otherwise, please flow it with the previous line. I'm not sure it's really necessary to document the ValueError though -- isn't it an invalid configuration state that shouldn't be encountered under normal conditions?